cluster-state-service/internal/features/e2einstancesteps/list_instances_steps.go

```diff
@@ -94,4 +100,28 @@ func init() {
 		}
 		exceptionList = append(exceptionList, Exception{exceptionType: exceptionType, exceptionMsg: exceptionMsg})
 	})
+
+	When(`^I try to list instances with redundant filters$`, func() {
+		url := "http://localhost:3000/v1/instances?cluster=cluster1&cluster=cluster2"
+		resp, err := http.Get(url)
+		if err != nil {
+			T.Errorf(err.Error())
+		}
+
+		exceptionList = nil
+		var exceptionType string
+		if resp.Status == badRequestHTTPResponse {
+			exceptionType = listInstancesBadRequest
+		} else {
+			T.Errorf("Unknown exception type '%s' when trying to list instances with redundant filters", resp.Status)
+		}
+
+		defer resp.Body.Close()
+		body, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			T.Errorf("Error reading expection message when trying to list instances with redundant filters")
+		}
+		exceptionMsg := string(body)
+		exceptionList = append(exceptionList, Exception{exceptionType: exceptionType, exceptionMsg: exceptionMsg})
```

Comment:
Yup. Its being verified in one of the subsequent steps - "And the instance exception message contains "At least one of the filters provided is specified multiple times"